Sources/_StringProcessing/Regex/DSLTree.swift

```diff
@@ -650,7 +680,7 @@ extension DSLTree.Node {
   /// output but forwarding its only child's output.
   var isOutputForwarding: Bool {
     switch self {
-    case .nonCapturingGroup:
+    case .nonCapturingGroup, .ignoreCapturesInTypedOutput:
```

Comment:
I wonder `wholeMatchType` would ever return the wrong type with this change.

It's probably not output-forwarding because it doesn't always have the same output as the child.